src/ServiceFabric.Logging/ServiceFabric.Logging.csproj

```diff
@@ -13,9 +13,9 @@
     <Product>ServiceFabric.Logging</Product>
     <AssemblyName>ServiceFabric.Logging</AssemblyName>
     <RootNamespace>ServiceFabric.Logging</RootNamespace>
-    <Version>2.0.0</Version>
+    <Version>2.0.1</Version>
```

Comment:
What do you think about leaving only the `Version` tag and deleting` AssemblyVersion` and `FileVersion`? it should be enough